examples/terminal_bench/test_basic.py

```
@@ -0,0 +1,145 @@
+        sys.modules['harbor.environments'] = MagicMock()
+        sys.modules['harbor.environments.base'] = MagicMock()
+        sys.modules['harbor.models'] = MagicMock()
+        sys.modules['harbor.models.agent'] = MagicMock()
+        sys.modules['harbor.models.agent.context'] = MagicMock()
+        
+        # Now try to import our agent
+        import praisonai_external_agent
+        print("✅ External agent import successful")
+        return True
+        
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)